### DISCUSSION TITLE
Getting too many error after statically built ffmpegkit

### DISCUSSION THREAD

**Discussion by hamza0786** [Udhayarajan/VidSnap#33] [General]
Hi Brother!
i am getting too many errors after i use the static ffmpeg library.plz help!

**Comment by Udhayarajan** [comment]
This issue is invalid and irrelevant to this repository. The repository goal is not to make it compatible with static FFmpeg. Could you contact me over email regarding this?

**Comment by hamza0786** [comment]
whats your mail?

**Comment by Udhayarajan** [comment]
Can you share the logs that you're getting?
